docs/operate/reference/viam-micro-server/micro-troubleshooting.md

```diff
@@ -1,25 +1,38 @@
 ---
-title: "viam-micro-server Troubleshooting"
-linkTitle: "viam-micro-server Troubleshooting"
+title: "Micro-RDK Troubleshooting"
+linkTitle: "Micro-RDK Troubleshooting"
 weight: 100
 no_list: true
 type: docs
 images: ["/installation/thumbnails/install.png"]
 imageAlt: "Install viam-micro-server"
-description: "Troubleshooting tips and best practices for installing and using viam-micro-server on a microcontroller."
+description: "Troubleshooting tips and best practices for installing and using viam-micro-server or other Micro-RDK-based firmware on a microcontroller."
 # date: "2024-10-07"
 # updated: ""  # When the content was last entirely checked
 ---
 
-For more `micro-rdk-installer` CLI usage options, see [GitHub](https://github.com/viamrobotics/micro-rdk/tree/main/micro-rdk-installer).
+## CLI for `viam-micro-server`
 
-For code examples and more information on `viam-micro-server`, see [GitHub](https://github.com/viamrobotics/micro-rdk).
+The Viam Micro-RDK installer is a command line tool that you can also use to troubleshoot.
+It allows you to:
 
-## Unstable connection
+- View logs locally and debug issues when your device has not connected to the Viam app
+- Change wifi credentials on your microcontroller
+- Overwrite firmware using a serial cable
 
-Because microcontrollers are more resource-constrained than single-board computers or other computers, when you run [control code](/dev/reference/sdks/) to control your `viam-micro-server` machine, you may experience instability.
+For more information on the `micro-rdk-installer` CLI usage options, see the [Micro-RDK Installer GitHub README](https://github.com/viamrobotics/micro-rdk/tree/main/micro-rdk-installer).
+
+## Code examples
+
+For code examples and more information on the Micro-RDK, see [the Micro-RDK GitHub repository](https://github.com/viamrobotics/micro-rdk).
+
```

Comment:
If this is a troubleshooting guide, I don't think examples are relevant here. This probably belongs elsewhere.

Reply:
Yeah...this link was already here but that's fair--will remove.